### ISSUE TITLE
Collapse "sameAs" links by default

### ISSUE THREAD

**Issue by esjewett** [humanitiesplusdesign/fibra#36]
Only display a single node for things that are "sameAs" each other.

**Comment by cncoleman** [comment]
Here we are assuming that identifying 5 instances of Francesco Algarotti as the same single person is a configuration/linked data magic task. There will be times when this reconciliation (or is it disambiguation, Eetu?) needs to happen in the CONSTRUCT  interface. Let's start by collapsing and add the new interface interaction later.

**Comment by esjewett** [comment]
Gah.

**Comment by esjewett** [comment]
All I really wanted to say here was that I agree, but I types something and then tried to click the cancel button, but that was actually the close button. Can someone at Github get on a UI that stops me from being incompetent?

**Comment by esjewett** [comment]
My strategy here is going to be to collapse these at the UI layer for the moment.

**Comment by jiemakel** [comment]
This should actually be a non-issue, as importing a new node from an external source already does the collapsing of the identities. 

However, we may have data right now where this doesn't hold, as the current set is a mishmash of random data pulled in with broken tools. And even if it weren't, the current getItemsForExplore -query is a horrible hack that pulls in information from multiple places where it shouldn't, further muddying the waters.

So in short, don't do anything about this on the UI level, I'll fix it in the data.

(The CONSTRUCT merging Nicole is talking about has also been on my shortlist of functionalities to do for quite a while.)

**Comment by esjewett** [comment]
Ah, ok. I will wait a bit then. Thanks!

**Comment by jiemakel** [comment]
I guess this problem has gone away at least for the moment now that I've cleaned our test data. 

We also shouldn't end up with multiple copies of the same identity anymore, as autocompletion does remote identity merging.

However, the next place where this _will_ come up is that getItems isn't currently merging identities of referenced remote objects, when it probably should.
